### PR TITLE
Update tutorial 

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
- include qpps
- fix queries/results

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-616.surge.sh

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 1]
```diff
@@ -549,16 +549,16 @@ For example, this query will find the nodes with `Person` label and the name `Ke
...
 
 .Result
 [role="queryresult",options="header,footer",cols="2*<m"]
 |===
-|+keanu.name+ | +keanu.born+
+| name | born
 
-| +"Keanu Reeves"+
-| +1964+
+| "Keanu Reeves"+
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]:
```suggestion
| "Keanu Reeves"
```

Comment (dogofbrian) [reply modules/ROOT/pages/introduction/cypher_tutorial.adoc]: Out of my purview, but noticed it.

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 2]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
 
 There are several ways in which Cypher can be used to search a graph for paths between nodes.
 
-To search for patterns of a fixed length, specify the distance ("hops") between the nodes in the pattern. 
-For example, the following query matches all `Person` nodes exactly 2 "hops" away from `Tom Hanks` and returns the first five rows. 
+To search for patterns of a fixed length, specify the distance ("hops") between the nodes in the pattern by using a xref::patterns/reference.adoc#quantifiers[quantifier] (`{}`).
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]: The empty braces by themselves might confuse, and are in any case not valid quantifiers. Maybe give the syntax in the form of `{n}`, or give the concrete quantifier `{2}` that fits the example.

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 3]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
...
 
 .Query
-[source, cypher, role=test-result-skip]
+[source, cypher]
 ----
-MATCH (tom:Person {name:'Tom Hanks'})-[*2]-(colleagues:Person)
-RETURN colleagues.name as colleagues LIMIT 5
+MATCH (tom:Person {name:'Tom Hanks'})-->{2}(colleagues:Person)
+RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
+ORDER BY bornIn LIMIT 5
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]:
```suggestion
MATCH (tom:Person {name:'Tom Hanks'})--{2}(colleagues:Person)
RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
ORDER BY bornIn LIMIT 5
```

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 4]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
...
+| "Penny Marshall" | 1943
 
-1+d|Rows: 5
+2+d|Rows: 5
 |===
 
 It is also possible to match a graph for patterns of a variable length. 
-The below query matches all `Person` nodes up to 3 "hops" away from `Tom Hanks` and returns the first five rows. 
-The xref:syntax/operators.adoc#syntax-using-the-distinct-operator[DISTINCT] operator ensures that the result contain no duplicate values. 
+The below query matches all `Person` nodes between `1` and `4` "hops" away from `Tom Hanks` and returns the first five rows, ordered by the `born` property value of the `colleagues` nodes in an ascending order.
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]:
```suggestion
The below query matches all `Person` nodes between `1` and `4` "hops" away from `Tom Hanks` and returns the first five rows, in ascending order of the `born` property value of the `colleagues` nodes.
```

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 5]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
 
 There are several ways in which Cypher can be used to search a graph for paths between nodes.
 
-To search for patterns of a fixed length, specify the distance ("hops") between the nodes in the pattern. 
-For example, the following query matches all `Person` nodes exactly 2 "hops" away from `Tom Hanks` and returns the first five rows. 
+To search for patterns of a fixed length, specify the distance ("hops") between the nodes in the pattern by using a xref::patterns/reference.adoc#quantifiers[quantifier] (`{}`). 
+For example, the following query matches all `Person` nodes exactly 2 "hops" away from `Tom Hanks` and returns the first five rows, ordered by the `born` property value of the `colleagues` nodes in an ascending order.
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]:
```suggestion
For example, the following query matches all `Person` nodes exactly 2 "hops" away from `Tom Hanks` and returns the first five rows, in ascending order of the `born` property value of the `colleagues` nodes.
```

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 6]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
...
+The below query matches all `Person` nodes between `1` and `4` "hops" away from `Tom Hanks` and returns the first five rows, ordered by the `born` property value of the `colleagues` nodes in an ascending order.
 
 .Query
 [source, cypher, role=test-result-skip]
 ----
-MATCH (tom:Person {name:'Tom Hanks'})-[*1..3]-(colleagues:Person)
-RETURN DISTINCT colleagues.name as colleagues LIMIT 5
+MATCH (p:Person {name:'Tom Hanks'})-->{1,4}(colleagues:Person)
+RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
+ORDER BY bornIn LIMIT 5
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]:
```suggestion
MATCH (p:Person {name:'Tom Hanks'})--{1,4}(colleagues:Person)
RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
ORDER BY bornIn LIMIT 5
```

Order by name as well if you want to guarantee the same order for everyone.

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc [hunk 7]
```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
...
+
 To find the shortest possible path between two nodes, use the `shortestPath` algorithm.
 For example, this query matches the shorest path in the graph between the two nodes `Tom Hanks` and `Keanu Reeves`:
 
 .Query
 [source, cypher]
 ----
 MATCH p=shortestPath(
-(keanu:Person {name:"Keanu Reeves"})-[*]-(tom:Person {name:"Tom Hanks"})
+(keanu:Person {name:"Keanu Reeves"})-[*]->(tom:Person {name:"Tom Hanks"})
```
Comment (dogofbrian) [inline modules/ROOT/pages/introduction/cypher_tutorial.adoc]: Are you sure about adding the direction? The diagram shows alternating relationship directions, so I can't see how that pattern would match.

**Review by dogofbrian — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/introduction/cypher_tutorial.adoc (hunk 7 above)
Comment (JPryce-Aklundh) [reply modules/ROOT/pages/introduction/cypher_tutorial.adoc]: No, you are correct. Thanks!

**Review by dogofbrian — APPROVED** [review]

**Pull request merged**